pgmpy/estimators/HillClimbSearch.py

```diff
@@ -158,6 +158,8 @@ def estimate(self, start=None, tabu_length=0, max_indegree=None):
 
             if best_operation is None:
                 break
+            if best_score_delta < epsilon:
```

Comment:
Got it. But isn't best_score_delta already the maximum positive change by which we can increase the score given the current network. So, it already is best_possible - curr ?